webapp/src/cardFilter.test.ts

```diff
@@ -10,59 +10,76 @@ import {TestBlockFactory} from './test/testBlockFactory'
 import {Utils} from './utils'
 
 import {IPropertyTemplate} from './blocks/board'
+import {blockTypes} from './blocks/block'
 
 jest.mock('./utils')
 const mockedUtils = mocked(Utils, true)
 
 const dayMillis = 24 * 60 * 60 * 1000
 
-describe('src/cardFilter', () => {
-    const board = TestBlockFactory.createBoard()
-    board.id = '1'
+const board = TestBlockFactory.createBoard()
+board.id = '1'
+
+const block = {
+    id: 'id',
+    boardId: 'boardId',
+    parentId: 'parnetId',
+    createdBy: 'createdBy',
+    modifiedBy: 'modifiedBy',
+    schema: 1,
+    type: blockTypes[13],
+    title: 'title',
+    fields: {
+        properties: {
+            propertyId: 'Status',
+        },
+    },
+    createAt: new Date('December 7, 2023').getTime(),
+    updateAt: new Date('December 7, 2023').getTime(),
+    deleteAt: new Date('December 7, 2023').getTime(),
+}
+const card = TestBlockFactory.createCardWithBlock(board, block)
 
-    const card1 = TestBlockFactory.createCard(board)
```

Comment:
This function relies on Date.now() to populate the `createAt` and `updatedAt` fields when a block is not provided. I created a new function `createCardWithBlock` and passed it a block stub to make it clearer how card fields are populated for this test suite.

Reply:
This makes sense, as at some point in time we might test cases for checking createAt, updateAt and other stuff.
Even though we can directly assign it to the card object the function makes more sense to avoid duplicity.